connection.go

```diff
@@ -207,25 +210,27 @@ type Opts struct {
 // Address could be specified in following ways:
```

Comment:
There are a lot of internal sctructures described in godoc, and I doubt many of them should be there

![image](https://user-images.githubusercontent.com/20455996/161789723-c05612de-9e9b-4411-bd92-dc9d6dadc66d.png)

Reply:
Added to #158.